mpqp/qasm/open_qasm_2_and_3.py

```diff
@@ -31,9 +31,10 @@
 
 import os
```

Comment:
We should update the header and the sphinx doc to incorporate all the work done for 3 to 2.

Reply:
done